MoltenVK/MoltenVK/API/vk_mvk_moltenvk.h

```diff
@@ -54,7 +54,7 @@ extern "C" {
 #define MVK_VERSION     MVK_MAKE_VERSION(MVK_VERSION_MAJOR, MVK_VERSION_MINOR, MVK_VERSION_PATCH)
 
 
-#define VK_MVK_MOLTENVK_SPEC_VERSION            7
+#define VK_MVK_MOLTENVK_SPEC_VERSION            8
```

Comment:
Spec version is already 8 (I beat you to a change yesterday!).

Bump this to 9, please.